packages/fiori/src/Timeline.ts

```diff
@@ -111,6 +188,116 @@ class Timeline extends UI5Element {
 			: Timeline.i18nBundle.getText(TIMELINE_ARIA_LABEL);
 	}
 
+	get growsOnScroll(): boolean {
+		return this.growing === TimeLineGrowingMode.Scroll;
+	}
+
+	get timeLineEndDOM(): Element {
+		return this.shadowRoot!.querySelector(".ui5-time-line-end-marker")!;
+	}
+
+	get growingButtonIcon() {
+		return this.layout === TimelineLayout.Horizontal ? "process" : "drill-down";
```

Comment:
If you use icons, you must import them.


The recommended way is to use the default export of the icons to make more explicit dependency to the used icon as the string constants and the imports completely disconnected:

```ts
import process from "@ui5/webcomponents-icons/dist/process.js";
import drillDown from "@ui5/webcomponents-icons/dist/drill-down.js";

class Timeline {


return this.layout === TimelineLayout.Horizontal ? process : drillDown;
```

Reply:
Done